action.yml

```diff
@@ -21,6 +21,10 @@ inputs:
     description: 'Location of your Pulumi files. Defaults to ./'
     required: false
     default: ./
```

Comment:
Don't we need to remove this default otherwise getInput will always get this default value?

Reply:
It seems so, yeah.

It might be better to remove the default on `work-dir`.

Following that change, we can deprecate `work-dir` but keep it as an alias to `working-directory` for backwards compatibility.